src/core-taggeds-failure/Failure/Failure.T/Failure.T.ToString.cs

```diff
@@ -6,6 +7,23 @@ partial struct Failure<TFailureCode>
 {
     public override string ToString()
         =>
-        Invariant(
-            $"Failure<{typeof(TFailureCode).Name}>:{{ \"FailureCode\": {FailureCode}, \"FailureMessage\": \"{FailureMessage}\" }}");
+        new StringBuilder(
+            "Failure<")
+        .Append(
+            typeof(TFailureCode).Name)
+        .Append(
+            ">:{ \"FailureCode\": ")
+        .Append(
+            FailureCode.ToString())
```

Comment:
Since there is `TFailureCode : struct` `where` constraint, the `FailureCode` may be a struct in general, not only a `enum`.
Thus, since a particular failure struct may implement [IFormattable](https://learn.microsoft.com/en-us/dotnet/api/system.iformattable), it would be better to use the `Invariant` (like it is implemented for the `SourceException` below).
```suggestion
            Invariant($"\"{FailureCode}\""))
```

Reply:
But. Please see https://github.com/pfpack/pfpack-core-taggeds/pull/127/files#r1288772355

Reply:
Fixed